### PR TITLE
OCPBUGS-33345: Disable create button on NAD empty state for regular user

### PR DESCRIPTION
**Pull request by upalatucci**
- disable button
- Change text using standards. Don't use `network attachment definition` but `NetworkAttachmentDefinition`

image

### CODE REVIEW

File: frontend/packages/network-attachment-definition-plugin/src/components/network-attachment-definitions/NetworkAttachmentDefinition.tsx [hunk 1]
```diff
@@ -136,7 +143,13 @@ const NADListEmpty: React.FC = () => {
   return (
     <EmptyState>
       <EmptyStateHeader
-        titleText={<>{t('kubevirt-plugin~No network attachment definitions found')}</>}
+        titleText={
+          <>
+            {t('kubevirt-plugin~No {{label}} found', {
```
Comment (hstastna) [inline frontend/packages/network-attachment-definition-plugin/src/components/network-attachment-definitions/NetworkAttachmentDefinition.tsx:148]: Shouldn't be plural used here instead? Originally, we use plural when showing something not found, in the list views.

Comment (upalatucci) [reply frontend/packages/network-attachment-definition-plugin/src/components/network-attachment-definitions/NetworkAttachmentDefinition.tsx:148]: yes you're right

**Review by metalice — APPROVED** [review]

**Review by hstastna — APPROVED** [review]

**Pull request merged**